train_diffusion_prior.py

```diff
@@ -237,26 +298,44 @@ def main():
     parser.add_argument("--save-path", type=str, default="./diffusion_prior_checkpoints")
 
     args = parser.parse_args()
-
-    print("Setting up wandb logging... Please wait...")
-
-    wandb.init(
-      entity=args.wandb_entity,
-      project=args.wandb_project,
-      config={
-      "learning_rate": args.learning_rate,
+    
+    config = ({"learning_rate": args.learning_rate,
       "architecture": args.wandb_arch,
       "dataset": args.wandb_dataset,
+      "Learning Rate":args.learning_rate,
+      "Weight Decay":args.weight_decay,
+      "Max Gradient Clipping Norm":args.max_grad_norm,
+      "Batch size":args.batch_size,
       "epochs": args.num_epochs,
+      "Diffusion_Prior_Network":{"dpn_depth":args.dpn_depth,
+      "dpn_dim_head":args.dpn_dim_head,
+      "dpn_heads":args.dpn_heads},
+      "Diffusion_Prior":{"dp_condition_on_text_encodings": args.dp_condition_on_text_encodings,
```

Comment:
let's use the same names as the constructor

Reply:
fixed